bundles/org.openhab.binding.noolite/src/main/java/org/openhab/binding/noolite/internal/NooliteMTRF64Adapter.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.noolite.internal;
+
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+
+import javax.xml.bind.DatatypeConverter;
+
+import org.openhab.binding.noolite.handler.NooliteMTRF64BridgeHandler;
+import org.openhab.binding.noolite.internal.config.NooliteBridgeConfiguration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import gnu.io.CommPortIdentifier;
+import gnu.io.SerialPort;
+import gnu.io.SerialPortEvent;
+import gnu.io.SerialPortEventListener;
+
+/**
+ *
+ * @author Petr Shatsillo - Initial contribution
+ *
+ */
+public class NooliteMTRF64Adapter {
+
+    private static Logger logger = LoggerFactory.getLogger(NooliteMTRF64Adapter.class);
+    DataInputStream in = null;
+    DataOutputStream out = null;
+    Thread watcherThread = null;
+    private static OutputStream output;
+    SerialPort serial;
+
+    public void connect(NooliteBridgeConfiguration config) throws Exception {
+        logger.debug("Opening serial connection to port {} with baud rate 9600...", config.serial);
+
+        CommPortIdentifier portIdentifier;
+        try {
+            portIdentifier = CommPortIdentifier.getPortIdentifier(config.serial);
+            serial = portIdentifier.open("openhab", 3000);
```

Comment:
Prefer to use a name that also includes you binding:
```suggestion
            serial = portIdentifier.open("org.openhab.binding.noolite", 3000);
```